docker-compose-db.yml

```
@@ -0,0 +1,131 @@
+version: "2.4"
+
+services:
+  db:
+    container_name: openimis-db
+    build:
+      context: https://github.com/openimis/openimis-db_dkr.git#${DB_BRANCH:-main}:/
+      args:
+        - ACCEPT_EULA=${ACCEPT_EULA}
+        - SA_PASSWORD=${DB_PASSWORD}
+        - SQL_SCRIPT_URL=${DB_SQL_SCRIPT}
+        - DB_SQL_SCRIPT_INIT_URL = ${DB_SQL_SCRIPT_INIT_URL}
+    networks:
+      openimis-net:
+        ipv4_address: 172.20.${IP_SUB:-20}.2
+# WARNING:
+# exposing the database port outside the openimis-net network
+# may lead to security issue (depending on your network topology)
+#    ports:
+#     - "1433:1433"
+  backend:
+    container_name: ${PROJECT_NAME:-openimis}-backend
+    build:
+      context: https://github.com/openimis/openimis-be_py.git#${BE_BRANCH:-main}:/
+      args:
+        OPENIMIS_CONF_JSON: ${OPENIMIS_BE_CONF_JSON}
+    command: start
+    environment: &backend-env
+      - DB_HOST=${DB_HOST}
+      - DB_PORT=${DB_PORT}
+      - DB_NAME=${DB_NAME}
+      - DB_USER=${DB_USER}
+      - DB_PASSWORD=${DB_PASSWORD}
+      - REMOTE_USER_AUTHENTICATION=False
+      - SITE_ROOT=api
+      - SITE_URL=${NEW_OPENIMIS_HOST}
+      - CELERY_BROKER_URL=amqp://localhost
```

to send the config at runtime too
```suggestion
      - CELERY_BROKER_URL=amqp://localhost
      - OPENIMIS_CONF_JSON=${OPENIMIS_BE_CONF_JSON}
```